src/index.js

```diff
@@ -6,16 +6,21 @@ import { makeServer } from "./server";
 import { store } from "app/store";
 import { Provider } from "react-redux";
 import { BrowserRouter } from "react-router-dom";
+import { myNewTheme, theme } from "styles/theme";
 import "./index.css";
 const rootElement = document.getElementById("root");
 const root = createRoot(rootElement);
 // Call make Server
 makeServer();
 
+console.log(theme.config.initialColorMode);
 root.render(
   <StrictMode>
     <Provider store={store}>
-      <ChakraProvider>
+      <ChakraProvider
+        theme={myNewTheme}
+        initialColorMode={theme.config.initialColorMode}
+      >
         <BrowserRouter>
```

Comment:
suggestion - browser router should be at the top

Reply:
It does not matter here as they are not related to each other.

Reply:
I think here it does not matter as they are not dependent on each other.